common/lib/capa/capa/inputtypes.py

```diff
@@ -325,12 +325,13 @@ def _get_render_context(self):
             'describedby_html': '',
         }
 
-        # Don't add aria-describedby attribute if there are no descriptions
-        if self.response_data.get('descriptions'):
-            description_ids = ' '.join(self.response_data.get('descriptions').keys())
-            context.update(
-                {'describedby_html': 'aria-describedby="{}"'.format(description_ids)}
-            )
+        status_id = 'status_' + self.input_id
+        descriptions = list([status_id])
```

Comment:
A comment about what you are doing here would be helpful. When trying to understand this code, it is confusing to set "descriptions" to a list containing a status_id.